indico/modules/designer/templates/template.html

```diff
@@ -14,6 +16,37 @@
 {% endblock %}
 
 {% block content %}
+    {% if tpls_count %}
+        <div class="affected-targets-warning weak-hidden">
+            {% call message_box('warning', fixed_width=true) %}
+                <p>
+                    {%- trans count=tpls_count, strong_start='<strong>'|safe, strong_end='</strong>'|safe,
+                              link_start='<a href="#" class="js-affected-objects">'|safe, link_end='</a>'|safe -%}
```

Comment:
IMO using set blocks would be cleaner, even if it makes the template a big longer. Especially in case we add more attributes in the future (kind of unlikely here...) this would quickly get messy.